spec/services/usps_in_person_proofing/enrollment_helper_spec.rb

```diff
@@ -75,60 +75,6 @@
         end
       end
 
-      context 'when the enrollment sponsor_id is not set' do
```

Comment:
Now that it will be impossible for an in_person_enrollment to be created with a nil sponsor_id, it makes sense to remove this context block. 👍🏻 LGTM